woocommerce/wc-admin
# Build: Move components to packages dir

This PR moves the components to a new package, `@woocommerce/components`, which will be published to NPM for 3rd party use. The components are also currently available on the `wc` global, ex: `wc.components.Card`.

The changes here are:
- The CSS for each component is now included in [a separate scss file](https://github.com/woocommerce/wc-admin/blob/update/packages-components/packages/components/src/style.scss), so that the build script picks up the CSS – the package build process doesn't handle extracting CSS from JS files, so we build these separately.
- File paths to other components are now using relative URLs, rather than relying on the components alias
- Chart components are left alone in `client/components`

**To test**

- Check that there is no functionality change in the app
- Test devdocs: `/wp-admin/admin.php?page=wc-admin#/devdocs`
- Note: there is an error in the filters example because the config objects passed in `packages/components/src/filters/example.md` are not up-to-date, that can be fixed in a following PR.
